process/sync/metablock.go

```diff
@@ -376,6 +377,20 @@ func (boot *MetaBootstrap) doJobOnSyncBlockFail(hdr *block.MetaBlock, err error)
 			log.Info(errNotCritical.Error())
 		}
 	}
+
```

Comment:
Just took a look on the function, and at line 370 "isForkDetected" - this variable name seems misleading, why is a fork detected when we have a timeout error? this seems not logical, so maybe use a better name for this.

Reply:
Agree. Renamed the variable